products/compute/api.yaml

```diff
@@ -7847,6 +7847,214 @@ objects:
+          the corresponding 'action' is enforced.
+        required: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'description'
+            description: |
+              A description of the rule.
+          - !ruby/object:Api::Type::Enum
+            name: 'versionedExpr'
+            description: |
+              Preconfigured versioned expression. For organization security policies rules,
+              the only supported type is "FIREWALL".
+            values:
+              - :FIREWALL
+            default_value: :FIREWALL
+          - !ruby/object:Api::Type::NestedObject
+            name: 'config'
+            description:
+              The configuration options for matching the rule.
+            properties:
+            - !ruby/object:Api::Type::Array
+              name: 'srcIpRanges'
+              description: |
+                Source IP address range in CIDR format.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::Array
+              name: 'destIpRanges'
+              description: |
+                Destination IP address range in CIDR format.
+              item_type: Api::Type::String
+            - !ruby/object:Api::Type::Array
+              name: 'layer4Config'
+              api_name: 'layer4Configs'
+              description: |
+                Pairs of IP protocols and ports that the rule should match.
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'ipProtocol'
+                    description: |
+                      The IP protocol to which this rule applies. The protocol
+                      type is required when creating a firewall rule.
+                      This value can either be one of the following well
+                      known protocol strings (tcp, udp, icmp, esp, ah, ipip, sctp),
+                      or the IP protocol number.
+                    required: true
+                  - !ruby/object:Api::Type::Array
+                    item_type: Api::Type::String
+                    name: 'ports'
+                    description: |
+                      An optional list of ports to which this rule applies. This field
+                      is only applicable for UDP or TCP protocol. Each entry must be
+                      either an integer or a range. If not specified, this rule
+                      applies to connections through any port.
+                      
+                      Example inputs include: ["22"], ["80","443"], and
+                      ["12345-12349"].
+      - !ruby/object:Api::Type::String
+        name: 'action'
+        description: |
+          The Action to preform when the client connection triggers the rule. Can currently be either
+          "allow", "deny" or "goto_next".
```

Comment:
I've added links to the proper documentation. `goto_next` is specifically documented [here](https://cloud.google.com/vpc/docs/using-firewall-policies#create-rules). Since the proto from Cloud Armor's non-org [SecurityPolicy](https://cloud.google.com/compute/docs/reference/rest/beta/securityPolicies) was recycled many fields are documented incompletely or incorrectly for OrganizationSecurityPolicy.